distribution/docs/src/main/jdocs/content/_configuring/managing-certificates-contents.adoc

```diff
@@ -83,28 +83,47 @@ To use the scripts, run them out of the `<INSTALL_HOME>/etc/certs` directory.
 
 For *NIX, use the `CertNew.sh` script.
 
-`sh CertNew.sh <FQDN>`
+`sh CertNew.sh <FQDN> [-san <tag:name,tag:name,...>]`
+
+where `<subject tag:name,tag:name,...>` represents optional subject alternative names to be added to the generated certificate. The format for subject alternative names is similar to the OpenSSL X509 configuration format. Supported tags are:
+
+* `email`
+* `URI` - uniformed resource identifier
+* `RID` - registered id
+* `DNS`
+* `IP` - ip address (V4 or V6)
+* `dirName` - directory name
 
 Alternatively, a distinguished name can be provided to the script with a comma-delimited string.
 
-`sh CertNew.sh -dn "c=US, st=California, o=Yoyodyne, l=San Narciso, cn=<FQDN>"`
+`sh CertNew.sh -dn "c=US, st=California, o=Yoyodyne, l=San Narciso, cn=<FQDN>" [-san <tag:name,tag:name,...>]`
```

Comment:
The example already provided for the distinguished name is not a fully concrete one. It still references <FQDN>. I didn't want to explode this section with now 4 examples (total combinations of all options) which is why I opted for documenting the SANs this way. What would you suggest?